diff --git a/code/julia-no-susume-20210115/print.jl b/code/julia-no-susume-20210115/print-debug.jl
index 150355f..3a88152 100644
--- a/code/julia-no-susume-20210115/print.jl
+++ b/code/julia-no-susume-20210115/print-debug.jl
@@ -1,11 +1,19 @@
+macro showln(ex)
+    ln = __source__.line
+    quote
+        print('L', $ln, ": ")
+        @show $ex
+    end |> esc
+end
+
 function foo(n)
     a, ret = 0, 0
     for i in 1:n
-        a += i
+        @showln a += i
         for j in 1:a
-            a += j
+            @showln a += j
         end
-        ret += a
+        @showln ret += a
     end
 
     return ret
