diff --git a/code/julia-no-susume-20210115/print.py b/code/julia-no-susume-20210115/print-debug.py
index 16c1163..9e46270 100644
--- a/code/julia-no-susume-20210115/print.py
+++ b/code/julia-no-susume-20210115/print-debug.py
@@ -4,10 +4,13 @@ def foo(n):
     a, ret = 0, 0
     for i in range(1,n+1):
         a += i
+        print(f"L6: a = {a}")
         for j in range(1,a+1):
             a += j
+            print(f"L9: a = {a}")
 
         ret += a
+        print(f"L12: ret = {a}")
 
     return ret
 
